arch/arm64/core/Kconfig

```diff
@@ -84,6 +84,14 @@ config HAS_ARM_SMCCC
 	  Include support for the Secure Monitor Call (SMC) and Hypervisor
 	  Call (HVC) instructions on Armv7 and above architectures.
 
+config ARM64_ENABLE_EL3_SERVICE
+	bool "To enable the ARM64 EL3 service"
+	default n
+	depends on HAS_ARM_SMCCC
+	help
+	  Enable Secure Monitor Call (SMC) handler in EL3 and enable
+	  SMC instructions at EL1 and above.
```

Comment:
What happens if you have already a secure monitor firmware running in EL3 and you enable this?

Reply:
You will use the Zephyr logic as Zephyr's vector table gets set to EL3 VBAR during initialization.

Reply:
Verified that if you enter Zephyr in EL3, Zephyr's vector table gets installed.  If someone has secure monitor firmware running in EL3, they enter Zephyr in EL3, and they have EL3 Services enabled then the SMC call will be handled by the EL3 Service SMC handler.